product_docs/docs/pgd/5/quick_start_aws.mdx

```diff
@@ -9,88 +9,91 @@ redirects:
-!!!
+    This set of steps is specifically for Ubuntu 22.04 LTS on Intel/AMD processors.  
+
 
 ### EDB account
 
-You'll need an EDB account in order to install both TPA *and* PGD. 
+You need an EDB account to install TPA and PGD. 
 
-[Sign up for a free EDB account](https://www.enterprisedb.com/accounts/register) if you don't already have one - this gives you a trial subscription to EDB's software repositories. 
+[Sign up for a free EDB account](https://www.enterprisedb.com/accounts/register) if you don't already have one. Signing up gives you a trial subscription to EDB's software repositories. 
 
-Once you have registered, or if you are already registered, go to the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page where you can obtain your "repo token". 
+After you're registered, go to the [EDB Repos 2.0](https://www.enterprisedb.com/repos-downloads) page to get your repo token. 
 
-On your first visit to this page you have to select **Request Access** to generate your "repo token". Copy the token using the Copy Token icon and store it safely. 
+On your first visit to this page, select **Request Access** to generate your repo token. Copy the token using the **Copy Token** icon, and store the token safely. 
 
 ### Install curl
 
-We use the `curl` command to retrieve installation scripts from repositories. On Ubuntu, this is not installed by default. To test if it is present, run `curl` in the terminal. 
+Use the `curl` command to retrieve installation scripts from repositories. On Ubuntu, this command isn't installed by default. To find out if it's installed, try running `curl` in the terminal: 
 
 ```console
 $ curl
 Command 'curl' not found, but can be installed with:
 sudo apt install curl
 ```
 
-If not found, run:
+If it's not found, install it:
+
 ```
 sudo apt -y install curl
 ```
 
 ### Setting environment variables
 
-First, set the `EDB_SUBSCRIPTION_TOKEN` environment variable to the value of your EDB repo token, obtained in the previous [EDB Account](#edb-account) step.
+Set the `EDB_SUBSCRIPTION_TOKEN` environment variable to the value of your EDB repo token that you got in the [EDB account](#edb-account) step:
 
 ```shell
 export EDB_SUBSCRIPTION_TOKEN=<your-repo-token>
 ```
 
-This can be added to your `.bashrc` script (or similar shell profile) to ensure it is always set.
+You can add this variable to your `.bashrc` script or similar shell profile to ensure it's always set.
 
 ### Configure the repository
 
-All the software needed for this example is available from the Postgres Distributed package repository. We download and run a script to configure the Postgres Distributed repository. This repository also contains the TPA packages.
+All the software needed for this example is available from the Postgres Distributed package repository. Download and run a script to configure the Postgres Distributed repository. This repository also contains the TPA packages.
```

Comment:
This sounds more like two steps than before....

The following command will download and run a script ... ?